product_docs/docs/migration_portal/4.0/04_mp_migrating_database/03_mp_schema_migration.mdx

```diff
@@ -66,17 +72,23 @@ You can migrate schemas to an existing on-premises EDB Postgres Advanced Server
 
```

Comment:
In the CLI and cmd/shell examples above, "<project_name>.sql" should be changed to something like "<sql_file_name>.sql" since there are more import file options now available.